semeio/jobs/correlated_observations_scaling/job.py

```diff
@@ -35,6 +37,20 @@ def scale(self, measured_data):
         matrix.std_normalization(inplace=True)
 
         scale_factor = matrix.get_scaling_factor(config.CALCULATE_KEYS)
+        events = config.CALCULATE_KEYS
+        data_matrix = matrix.get_data_matrix()
+        nr_components, singular_values = DataMatrix.get_nr_primary_components(
+            data_matrix, threshold=events.threshold
+        )
+        self._reporter.publish("COS.svd", list(singular_values))
```

Comment:
The reporter is already namespace with the name of the workflow; should we perhaps do `svd` only?